partcad/src/partcad/wrappers/wrapper_render_3mf.py

```
@@ -0,0 +1,54 @@
+        cq.exporters.export(
+            cq_solid,
+            path,
+            tolerance=request["tolerance"],
+            angularTolerance=request["angularTolerance"],
+        )
+
+        return {
+            "success": True,
+            "exception": None,
+        }
+    except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)